jest.integration.config.ts

```diff
@@ -8,7 +8,7 @@ const config: Config.InitialOptions = {
   testRunner: "jasmine2",
   globalSetup: "./src/handlers/int-test-support/helpers/testSetup.ts",
   verbose: true,
-  testTimeout: 1600000,
+  testTimeout: 300000,
```

Comment:
Description of this PR says you're increasing timeout, but here you're decreasing -- just wanted to make sure this value isn't a mistake.

Reply:
earlier testTimeout was wrong it was supposed to be 160000 now this has been changed to 300000 ms